### PR TITLE
Set the default clock value within FakeAsync.run()

### PR DESCRIPTION
**Pull request by nex3**
This brings this package to full API compatibility with fake_async
0.1.

### CODE REVIEW

File: lib/fake_async.dart [hunk 1]
```diff
@@ -110,15 +129,24 @@ class FakeAsync {
   /// asynchronous features used within [callback] are controlled by calls to
   /// [elapse] rather than the passing of real time.
   ///
+  /// The [`clock`][] field will be set to a clock that reports the fake elapsed
```
Comment (natebosch) [inline lib/fake_async.dart]: [nit] I think we'd typically call this a 'top-level variable' instead of a 'field'

Comment (natebosch) [inline lib/fake_async.dart]: We sometimes do normal dart-doc references across packages. It works well for in-editor navigation. Does it not work for rendered dartdoc navigation?

**Review by natebosch — APPROVED** [review]

File: lib/fake_async.dart (hunk 1 above)
Comment (nex3) [reply lib/fake_async.dart]: Doesn't look like it. See for example [`SourceSpan.message` here](https://www.dartdocs.org/documentation/yaml/2.1.13/yaml/YamlNode/span.html).

Comment (nex3) [reply lib/fake_async.dart]: Dartdoc calls them "properties", so I'll use that.

**Pull request merged**